client/lib/domains/types.ts

```diff
@@ -101,6 +101,7 @@ export type ResponseDomain = {
 	registrar: string;
 	registrationDate: string;
 	renewableUntil: string;
+	showTransferConnectedDomainNudge: boolean | null;
```

Comment:
I think this could be boolean only without problem, because `assembler.js `force a boolean value.